build/scormAPI.js

```diff
@@ -0,0 +1,935 @@
+/*! simplify-scorm v1.0.1 - 2018-04-09 */

```

Comment:
Thinking the build folder should not be included

Reply:
GitHub has an option to attach a binary as part of the official release, I am pretty sure we can use this to attach the minified final output to a release, but I will also verify this

Reply:
Ah, yes, I see that now. Removing the build folder.